esx_service/utils/auth_data.py

```diff
@@ -410,7 +422,8 @@ def create_default_tenant(self):
                                            name=auth.DEFAULT_TENANT, 
                                            description="This is a default tenant", 
                                            vms=[], 
-                                           privileges=[])
+                                           privileges=[],
+                                           tenant_uuid=auth.DEFAULT_TENANT_UUID)
```

Comment:
I don't agree, because if we ever need to use any special tenant uuid, we will always introduce corresponding special tenant name. So the uuid handling can always be encapsulated in the method itself.

Anyway, I will let you make the decision since this is not really a big concern.